server/test/user.spec.js

```diff
@@ -101,6 +102,32 @@ describe('Users Routes', () => {
       expect(res).to.be.a('object');
       expect(res.body).to.have.property('data');
     });
+
+    it('should not retrieve users functionality if authorization token is not valid', async () => {
+      const res = await chai.request(app)
+        .get('/api/v1/users/')
+        .set({ Authorization: `Bearer ${invalidToken}` });
+      expect(res).to.have.status(401);
+      expect(res).to.be.a('object');
+      expect(res.body).to.have.property('message').eql('Error in verification. Please try again');
+    });
+    it('should retrieve users functionality', async () => {
```

Comment:
@geekdanitek Kindly give more insight into this

Reply:
`/users` list all users functionality (profile)